src/main/java/uk/ac/ebi/ena/webin/cli/upload/AbstractRetryCapableUploadService.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ebi.ena.webin.cli.upload;
+
+import org.springframework.retry.support.RetryTemplate;
+import org.springframework.retry.support.RetryTemplateBuilder;
+
+import java.util.stream.Stream;
+
+public abstract class AbstractRetryCapableUploadService implements UploadService {
+
+    protected RetryTemplate createRetryTemplate(Class<? extends Exception>... retryOnErrors) {
+        RetryTemplateBuilder builder = RetryTemplate.builder()
+            .maxAttempts(4)
+            .exponentialBackoff(1_000, 5, 10_000); // 1, 5, 10
```

Comment:
- We could wait a bit longer possibly e.g up to even 1 minute.
- I wonder if we should log retry attempts?
- If we want to log retry attempts then we need to pass in a message to the createRetryTemplate e.g.:
```Retrying to make a connection to the webin.ebi.ac.uk FTP```